dotCMS/src/integration-test/java/com/dotcms/datagen/ContainerAsFileDataGen.java

```diff
@@ -20,16 +20,32 @@
 import java.util.Arrays;
 import java.util.List;
 import java.util.Set;
+import java.util.stream.Collectors;
 
 public class ContainerAsFileDataGen extends AbstractDataGen<FileAssetContainer> {
 
+    private class ContentTypeContent {
+        private ContentType contentType;
+        private String content;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Private field 'content' could be made final; it is only initialized in the declaration or constructor.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5067852)